content/competitions/hackathon-2022.html

```diff
@@ -0,0 +1,110 @@
+        <p class="subtitle is-6">49-301, Advanced Engineering Building.</p>
+
+        <div class="content">
+            <div class="buttons has-text-centered">
+                <a href="https://events.humanitix.com/uqcs-hackathon-2022" class="button is-link uqcs-blue">Get your tickets now!</a>
+                <a href="https://www.facebook.com/events/383019433815672/" class="button">Facebook event</a>
+            </div>
+
+			<p>
+                Form your teams, brainstorm your cool ideas, and get ready to eat tons of free food... UQCS Hackathon 2022 is here!! Featuring prizes bigger and better than ever, SEVEN free meals, and 24 hour stay (TBD)! That's right - you'll be able to stay at uni overnight to get that authentic hackathon experience (TM).
+			</p>
+
+			<p>
+                Any year and experience level is welcome! In the past, students have built websites, desktop apps, games, hardware, and even robots and if you've never been to a hackathon before, not to worry - we'll be hosting a talk before hackathon weekend to walk you through and give you tips and tricks for your first hackathon experience.
+			</p>
+
+			<p>
+                <b>EARLY BIRD TICKETS</b>: student tickets will only be $10 if you purchase one <b>on or before 5th Aug</b>, or only $15 with a <i>limited edition laptop blahaj</i> shirt included! Other ticket types can be found either below or at the Humanitix event link.
+			</p>
+
+            <h3>Projects</h3>
+            <p>
+                Some past projects are on our showcase page.
+            </p>
+            <p>
+                <a class="button" href="/showcase/">UQCS Showcase</a>
+            </p>
+
+            <h3>Pricing</h3>
+
+            <p>
+                Early bird student tickets are just $10 (or $15 including a limited edition t-shirt). Otherwise, tickets are $15 for students and $20 for non-students (non-students are not eligible for prizes). As usual, we
+                will be supplying breakfast, lunch and dinner during the Hackathon too - make sure to grab your tickets so we can know how many people to cater for. See you there!
+            </p>
+
+            <h3>Prizes</h3>
+
+            <p>We'll have <b>$5500</b> in prizes up for grabs across ten categories:</p>
+
+            <h5>$2000 - Best Overall</h5>
+            <p>The creators of the best project (as determined by our celebrity judges) will take home a whopping $2000
+                dollars (not to mention bragging rights)! This prize is proudly sponsored by TechnologyOne.
+            </p>
+
+            <h5>$1000 - Best Overall Runner Up</h5>
+            <p>The second placed team (as determined by our celebrity judges) will take home $1000!
+            </p>
+
+            <h5>$500 - Best Newcomers</h5>
+            <p>The best team consisting entirely of students in their first or second semester of university (in any discipline) will find themselves the proud owners of $500 to distribute between themselves.
```

Comment:
Perhaps a note that this isn't hard-and-fast and the final decision of whether a team is eligible for the first-year-prize will be made by committee on the day?

Reply:
Also "Best Newcomers" is a good title but it sorta implies new-to-this, not "first-year"

Reply:
Fair point, will change to "Best First-Years"

Reply:
_Why not "Best Neophytes"?_